ROVU/Core/src/utility/TestSuite.java

```diff
@@ -31,20 +46,20 @@
 	
 	Set<RobotHandler> testSet = new HashSet<>();
 	
-	AbstractSimulatorMonitor testController;
+	AbstractSimulatorMonitor <RobotHandler>testController;
 	
-	  @Before
```

Comment:
It seems like this is the correct way to do it? To initialise the members in a @Before setUp() method? Then you can use `testSubject` in the other methods.

Reply:
With @before it starts simulator every time the test case is run, slowing everything down and causing enormous lag. Also, I create completely separate variables for all tests.